specs/core/0_beacon-chain.md

```diff
@@ -946,7 +946,7 @@ def get_attestation_participants(state: BeaconState,
     participants = []
     for i, validator_index in enumerate(crosslink_committee):
         aggregation_bit = get_bitfield_bit(bitfield, i)
-        if aggregation_bit == 0b1:
+        if aggregation_bit == 1:
```

Comment:
Why change from `0b1` to `1`? Making it clear that it's a bit with the `0b` prefix seems like a win to me.

Reply:
Can debate @vbuterin when he gets here.
I lean toward `0b` prefix but don't have a dog in this fiht.